Library/Formula/facebook-flint.rb

```diff
@@ -0,0 +1,36 @@
+class FacebookFlint < Formula
+  desc "Flint - An open-source lint program for C++ developed by, and used at Facebook."
+  homepage "https://github.com/facebook/flint"
+  url "https://github.com/facebook/flint.git"
```

Comment:
If there's no stable release you'll need to file this at the `head-only` tap. Otherwise, please use a tag or tagged tarball and SHA256.